packages/module/src/styles/style.scss

```diff
@@ -1,4 +1,9 @@
 @import '~@patternfly/patternfly/sass-utilities/all';
+@import '~@patternfly/patternfly/utilities/Text/text.scss';
```

Comment:
If you use the PF react components these should not be needed

Reply:
Tried removing them to no avail, instead importing addons, which also imports sass-utilities